db/migrate/20200831144533_create_trips.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :trips do |t|
       t.references :origin, null: false, foreign_key: { to_table: :countries }
       t.references :destination, null: false, foreign_key: { to_table: :countries }
-      t.references :user, null: false, foreign_key: true
+      t.references :user, null: true, foreign_key: true
```

Comment:
Why are we allowing trips to not have users?

Reply:
because you also want the app to work for visitors. See above